sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getCompletedApplicationDetails.e2e-spec.ts

```diff
@@ -268,6 +269,50 @@ describe("ApplicationStudentsController(e2e)-getCompletedApplicationDetails", ()
       });
   });
 
+  it("Should get application details when the current assessment is impacted and the trigger type is 'Related application changed'.", async () => {
+    // Arrange
+    const application = await saveFakeApplicationDisbursements(
+      appDataSource,
+      { student },
+      {
+        applicationStatus: ApplicationStatus.Completed,
+        createSecondDisbursement: true,
+        currentAssessmentInitialValues: {
+          triggerType: AssessmentTriggerType.RelatedApplicationChanged,
+        },
+      },
+    );
+
+    const [firstDisbursement, secondDisbursement] =
+      application.currentAssessment.disbursementSchedules;
+    firstDisbursement.coeStatus = COEStatus.completed;
```

Comment:
Not a blocker, in my previous comment I mentioned about using initital values like this. 

![image](https://github.com/bcgov/SIMS/assets/54600590/fc7727db-41cc-4636-ab1b-b8d9c979d7db)

Reply:
when factory is used, following piece of code is not required.

```ts
application.applicationStatus = ApplicationStatus.Completed;
    application.currentAssessment.triggerType =
      AssessmentTriggerType.RelatedApplicationChanged;
    await applicationRepo.save(application);
```